lib/cache/PackFileCacheStrategy.js

```diff
@@ -768,6 +768,14 @@ class PackContent {
 	}
 }
 
+const allowCollectingMemory = buf => {
+	const wasted = buf.buffer.byteLength - buf.byteLength;
+	if ((wasted > 8192 && wasted > 1048576) || wasted > buf.byteLength) {
```

Comment:
hi @sokra - are these intended to both be greater than checks?

Reply:
Yeah good find. The brackets should be around the ||